WeatherMonitor/InputData/JsonInput.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Text.Json;
+using WeatherMonitor.PrintConfig;
+using WeatherMonitor.WeatherManagement;
+namespace WeatherMonitor.InputData;
+
+public class JsonInput : IInputData
+{
+    public WeatherData? GetDate(string input)
+    {
+        WeatherData? weatherData = null;
+        try
+        {
+            weatherData = JsonSerializer.Deserialize<WeatherData>(input);
+        }
+        catch
+        {
+            Print.Log("Invalid Json Format");
+        }
+        return weatherData;
+    }
+
+    public bool IsMatch(string input)
+    {
+        return input.StartsWith('{') && input.EndsWith('}');
```

Comment:
Question: why not using this 
```suggestion
        return input.StartsWith("{") && input.EndsWith("}");
```

Reply:
case I see ``` StartsWith('{') ``` is more appropriate because I am checking single character in the string

Reply:
it is faster sending char instead of a string